### DISCUSSION TITLE
List of Languages

### DISCUSSION THREAD

**Discussion by kuyaC** [dlutton/flutter_tts#191] [Q&A] [answered]
Hello,
where can I find a complete list of languages&dialects/accents/locales (you know what I mean) that are supported by this tts engine?

Thanks :)
-Cedric

**Comment by dlutton** [comment, marked as answer]
This plugin uses the platform specific API for TTS. Android uses the google text to speech package (com.google.android.tts) and iOS uses Apple's text to speech package.  For Android, the supported languages depends on the Android API version on your device and what that supports.  There is no complete list because it varies.

**Reply by echogit** [reply]
Does this have changed, I mean, maybe it is more stable and might be a common set on languages/voices available as default?
I'm having a bad time trying to select an usable voice for iOS without having to ask for the user to download it.
